p2p/ipld/validate.go

```diff
@@ -0,0 +1,99 @@
+package ipld
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	format "github.com/ipfs/go-ipld-format"
+	coreiface "github.com/ipfs/interface-go-ipfs-core"
+	"github.com/lazyledger/nmt/namespace"
+
+	"github.com/lazyledger/lazyledger-core/types"
+)
+
+// ValidationTimeout specifies timeout for DA validation during which data have to be found on the network,
+// otherwise ErrValidationFailed is thrown.
+// TODO(@liamsi): define proper value
```

Comment:
Better to capture this in an issue too.

Reply:
https://github.com/lazyledger/lazyledger-core/issues/280